lib/fund.js

```diff
@@ -56,7 +58,15 @@ class Fund extends BaseCommand {
     this.fund(args).then(() => cb()).catch(cb)
   }
 
-  async fund (args) {
+  execWorkspaces (args, filters, cb) {
```

Comment:
isn't this effectively what the new PR for all the reify commands does?

Reply:
yeah great point, let's bring that class into this PR!